app.js

```diff
@@ -19,7 +19,30 @@ app.use(express.static(path.join(__dirname, 'public')));
 // Add the route handlers here:
 
 app.get('/', (req, res) => {
-  res.render('index');
+  res.render('index.hbs');
 });
 
-app.listen(3000, () => console.log('🏃‍ on port 3000'));
+
+app.get('/beers', (req, res) => {
+  //res.render(__dirname + '/views/beers.hbs', ) // how it was before
+  punkAPI.getBeers()
+    .then(beersFromApi => 
+      {console.log('Beers from the database: ', beersFromApi)
```

Comment:
watch code cleanness, on multiline statements the opening curly bracket should start in the line above.